plugin/metrics/prometheus/metricsstore/reader.go

```diff
@@ -26,14 +26,15 @@ import (
 	"time"
 	"unicode"
 
-	"github.com/opentracing/opentracing-go"
```

Comment:
can we undo changes to this file? I think we decided not to touch it in this PR

Reply:
Yes, sure I am.